### ISSUE TITLE
History saved on google search box.

### ISSUE THREAD

**Issue by And96** [kiwibrowser/src#75]
A small privacy issue.

When I search something on google, it remember everything I searched in the search bar.

There is an option inside google settings to disable history. It works fine in others browsers but oddly doesnt in Kiwi.

Screenshot here:
https://m.imgur.com/a/hf88u0b

**Comment by SuperSandro2000** [comment]
![](https://i.imgur.com/XzyJEuH.jpg)

**Comment by rururux** [comment]
Anyway, you can disable search history in search settings page... but before that you need to sign out.
(cuz this settings only show when you not signed in)
after the setting, it will keep even you sign in.
![Screenshot_20200501-234039~01](https://user-images.githubusercontent.com/5568054/80814195-63c88700-8c06-11ea-9f9d-8ca15fc729d8.png)

**Comment by kiwibrowser** [comment]
Possible cause (not sure) is that Google is tracking search history by IP address: https://www.google.com/complete/search?client=chrome&gs_ri=chrome-mobile-ext-ansg&xssi=t&oit=1&cp=1&pgcl=4&gs_rn=42&psi=-vvd1pukpHuVUAfU&sugkey=KIWIBROWSER&q=a

(example here)

Replace &q=a with some other queries and try on another device and you can see some past navigation

This is not specific to Kiwi though

**Comment by And96** [comment]
@kiwibrowser No. I cannot see the history if I copy the URL on a different device.

@rururux It's a workaround but it work! 
In the account I have disabled history, but it seems it is ignored by kiwi or google. That's really strange.

It doesn't happen on firefox.

I noticed a different URL, I don't know if it can makes some differences:
https://www.google.com/search?q=SEARCH%20HERE&ie=utf-8&oe=utf-8&client=firefox-b-m
